syslog-ng.service: pid file location to match pmon.d

pmon.d expects syslog-ng pid file to be at location
/var/run/syslog-ng/syslog-ng.pid.

diff --git a/contrib/systemd/syslog-ng.service b/contrib/systemd/syslog-ng.service
index e724363..606a967 100644
--- a/contrib/systemd/syslog-ng.service
+++ b/contrib/systemd/syslog-ng.service
@@ -5,7 +5,8 @@ Documentation=man:syslog-ng(8)
 [Service]
 Type=notify
 Sockets=syslog.socket
-ExecStart=/usr/sbin/syslog-ng -F -p /var/run/syslogd.pid
+ExecStartPre=-/usr/bin/mkdir -p /var/run/syslog-ng/
+ExecStart=/usr/sbin/syslog-ng -F -p /var/run/syslog-ng/syslog-ng.pid
 ExecReload=/bin/kill -HUP $MAINPID
 StandardOutput=null
 Restart=on-failure
